conf/gulp-tasks/template/bundle.js

```diff
@@ -0,0 +1,79 @@
+const { dest } = require('gulp');
+
+const fs = require('fs');
+const insert = require('rollup-plugin-insert');
+
+const rollup = require('gulp-rollup-lightweight');
+const babel = require('rollup-plugin-babel');
+const resolve = require('rollup-plugin-node-resolve');
+const commonjs = require('rollup-plugin-commonjs');
+const builtins = require('rollup-plugin-node-builtins');
+
+const source = require('vinyl-source-stream');
+
+/**
+ * @param {Function} callback called when the stream ends
```

Comment:
Is this callback invoked when the bundleTemplates task is done? I'm not sure which stream is being referred to.